libyul/AsmAnalysis.cpp

```diff
@@ -68,6 +68,8 @@ bool AsmAnalyzer::analyze(Block const& _block)
 	auto watcher = m_errorReporter.errorWatcher();
 	try
 	{
+		// FIXME: Pass location of the object name. Now it's a location of first code section in yul
+		validateObjectStructure(_block.debugData->nativeLocation);
```

Comment:
`debugData` is not always present. It may be `null`. You need a check against this. Or just use the helper that does that for you:
```suggestion
		validateObjectStructure(nativeLocationOf(_block));
```